addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -102,6 +102,7 @@ class Applicant(models.Model):
         ('refused', 'Refused'),
         ('archived', 'Archived'),
     ], compute="_compute_application_status")
+    just_unarchived = fields.Boolean()
```

Comment:
Adding a new field is a bit overkill IMO, instead you can use a context key in toggle_active and then in `_track_template` you can check for this context key:

```py
    def toggle_active(self):
        self = self.with_context(blabla=True)
        [...]
```

Reply:
That was what I did first :innocent: 

But, `_track_template` is called from `_message_track_post_template` and just before that, the context is cleaned, so that won't work :/

Reply:
@kba-odoo, indeed, I overcomplicated stuff :innocent: